### PR TITLE
add first csp rule template

### PR DESCRIPTION
**Pull request by CohenIdo**
First CSP rule template, working with the following Kibanaa PR:
- https://github.com/elastic/kibana/pull/130120

### CODE REVIEW

**Comment by oren-zohar** [comment]
@CohenIdo can you please give some context?

**Comment by CohenIdo** [comment]
@oren-zohar it's still a draft, sorry about the confusion

**Comment by CohenIdo** [comment]
@oren-zohar it is still a draft sorry for the confusion

**Review by kfirpeled — APPROVED** [review]
lgtm as a starting point

File: packages/cis_kubernetes_benchmark/kibana/csp_rule_template/csp_rule_template-9129a080-7f48-11ec41308bcdaaf665761478bb6f0d745a5c.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        "remediation": "Run the below command (based on the file location on your system) on the\nmaster node.\nFor example,\n```\nchmod 644 /etc/kubernetes/manifests/kube-apiserver.yaml\n```\n",
+        "benchmark": {
+            "name": "CIS Kubernetes V1.20",
+            "version": "v1.0.0"
+        },
+        "enabled": true,
+        "muted": false,
+        "rego_rule_id": "cis_1_2_2"
+    },
+    "id": "csp_rule_template-41308bcdaaf665761478bb6f0d745a5c",
```
Comment (eyalkraft) [inline packages/cis_kubernetes_benchmark/kibana/csp_rule_template/csp_rule_template-9129a080-7f48-11ec41308bcdaaf665761478bb6f0d745a5c.json:24]:
```suggestion
    "id": "csp_rule_template-9129a080-7f48-11ec41308bcdaaf665761478bb6f0d745a5c",
```
or rename the file to
```
packages/cis_kubernetes_benchmark/kibana/csp_rule_template/csp_rule_template-41308bcdaaf665761478bb6f0d745a5c.json
```

**Comment by eyalkraft** [comment]
Also the PR name is Add 2 rule assets but there's only one 😄

**Review by eyalkraft — APPROVED** [review]

**Pull request merged**